### ISSUE TITLE
unibeautify:disable deletes line of code

### ISSUE THREAD

**Issue by Glavin001** [Unibeautify/unibeautify#47]
See [Playground](https://playground.unibeautify.com/#N4IgNghgdg5grhGBTAchAtkkAuEApCANwgGUBjAJwEsAHAFxABoQB7eqlqAZx1AGEAxLxAAjJBDh0qAMypIKPbAG0QABQpI6dAJ4ACACIzpIALoBfZnxIlhYiVNnzFK9Zp0GjTNRq1yKXvBIAWgAhcUkZbVMLECsANVtwhz9nbzc9Q2ljc0sWMAATADE4Lg4oRPsZFJwXH3dM7JiAUUCKiMcFGrStDM9mQNCkyOjmAHEKCBoACwBFABk25Kcu1195EZAACQAVAFkF7FA7durlbvq+-GCwyukonK29uYBqJoAlEMWq5bPVi6yNptoPkwEgRBBOodREMOqk-r0Af1rjD7jFAgB5FBfWErOoI4zMP5+ALI26opEADWxp1q6Q8iPOxOYTRIcyoUAYSMGZI2BGI1J+tJ69Ma-SIpEotAYUOOS0hQv+BMZ62ZrPZnKu3Pa5JAcyQXEURxhNPO+K8RPWDzZAEc4FR8gL5ZqbtqNrsIQBrfIsADu5RlxsFyv8D12JToEDIUywAduJoGLocOreHGlRrjQfhIo2JCsNljJ0zeOzhLqxIeJDio0dcOLDRzEANNdxdPrFZokbBEMN0IzTqzbZiJB9VBgzd+dc8D22VAjUCocHQuhIOlBy+jmh7su+-cnAOnI7HBbltdbU5i220NCQ5Go9HHCrNpbc5ZicSoXAQYGkLAoZBj6aFruZ77m+cAAb2QGnmsIYxBS+wPqaJbOiiGwUtsSDoDQkB0BB244hOIGNDEv6juyEBgBhAAe0ogNIcBQGQUicLouFcHQAAUACUujAAAOlAuhCboAD0Im6AxVD4do2D5B+EAiKCAlkJwXB5EgAB0YAsDAHF8SAGHsQAjPpXEANwCcJoniZJ0myfJilIMpqnqVpOl6QZ+p0AATKZFlQGYAkCfRjHMYJ0ZgNpADqv4FNxvFBXQYkSfOdlyVwClKYxLmgm5ukAOSbEgkUsLoMUUAU+XmYlyW2Si9kZY5zncK52kFUVJVlbF+RVf5ZggGYQA)

Before:

```javascript
function test() {
    // unibeautify:disable
console.log("Test1");
    // unibeautify:disable
console.log("Test2");
}

function helloWorld() {
	// unibeautify:disable
console.log('Hello World');
	// unibeautify:disable
console.log('Hello World');
}
```

After:

```javascript
function test() {
    // unibeautify:disable
console.log("Test2");
}

function helloWorld() {
	// unibeautify:disable
console.log('Hello World');
	// unibeautify:disable
console.log('Hello World');
}
```

**Comment by Glavin001** [comment]
Related to https://github.com/Unibeautify/unibeautify/issues/46

**Comment by Glavin001** [comment]
Related to https://github.com/Unibeautify/unibeautify/issues/39

**Comment by Glavin001** [comment]
Could not reproduce after fixing #46 . See https://github.com/Unibeautify/unibeautify/pull/51

**Comment by Glavin001** [comment]
Published to v0.9.3
